### DISCUSSION TITLE
Page build failure and no update on the site

### DISCUSSION THREAD

**Discussion by artemmironchik** [community/community#35725] [Other Feature Feedback, Questions, & Ideas]
Hello,

I have a website on GitHub Pages (https://rolling-scopes-school.github.io/artemmironchik-JSFE2022Q3/online-zoo/pages/main/). I modified my files and pushed changes, but there are no changes on the site and I received email with page build failure. What should I do ?

Thanks in advance

**Comment by airtower-luna** [comment]
Check the build log via the Actions tab in the repository to see what might be wrong. I tried to open the repository (not the deployed page) to take a look, but seems to be private, so I'm afraid "check the log" is about all that can be said from outside. :sweat_smile:

**Reply by artemmironchik** [reply]
Thanks for the answer, but I’ve already fixed it just by merging new pull request with changes that were made after failed one.
